src/main/java/http/server/handlers/GetHandler.java

```diff
@@ -0,0 +1,12 @@
+package http.server.handlers;
+
+import http.server.Handler;
+import http.server.Response;
+
+public class GetHandler implements Handler {
+    public void setResponseValues(Response response) {
+        response.setStatusCode(200, "OK");
```

Comment:
Curious - when you were deciding how the response will be built (eg. with a group of setters like in your case), did you consider building it by passing all these values as arguments at creation?